kitchen.yml

```diff
@@ -7,43 +7,27 @@ driver:
 driver_config:
```

Comment:
This line and the blank line above can be removed.

Reply:
true but it doesn't hurt. :)